packages/dds/tree/src/feature-libraries/sequence-field/sequenceFieldToDelta.ts

```diff
@@ -68,7 +69,8 @@ export function sequenceFieldToDelta<TNodeChange>(
                     out.pushContent(deleteMark);
                     break;
                 }
-                case "MoveOut": {
+                case "MoveOut":
+                case "ReturnFrom": {
                     const moveMark: Delta.MoveOut = {
```

Comment:
We need to deal with changes here if any. I'm guessing this will happen in a separate PR. Feel free to resolve/ignore without further action.